src/printer.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function Printer (listOfTransactions) {
+  this.listOfTransactions = listOfTransactions
+}
+
+Printer.prototype.formatTransactionLine = function (transaction) {
+  if (!transaction.credit) { transaction.credit = '' }
```

Comment:
I'm worried that the code to print a transaction ends up destroying some of the data on the transaction model in the process of printing it.